tests/pipelines/test_register_step.py

```diff
@@ -17,7 +21,19 @@
 )  # pylint: enable=unused-import
 
 
-@pytest.mark.usefixtures("clear_custom_metrics_module_cache")
+@pytest.fixture
+def registry_uri_path(tmp_path):
+    previousRegistryUri = mlflow.get_registry_uri()
+    db_tmpdir = tempfile.mkdtemp()
+    path = os.path.join(db_tmpdir, "registry.db")
+    db_url = "sqlite:///%s" % path
+    SqlAlchemyStore(db_url, db_tmpdir)
+    yield db_url
+    print("Clean Up Path", db_url)
+    shutil.rmtree(db_tmpdir)
+    mlflow.set_registry_uri(previousRegistryUri)
```

Comment:
Can we run these in the `finally:` block of a `try/finally` to make sure this runs even if the test case throws an exception?